libclc/CMakeLists.txt

```diff
@@ -42,6 +42,9 @@ set( LIBCLC_TARGETS_TO_BUILD "all"
 
 option( ENABLE_RUNTIME_SUBNORMAL "Enable runtime linking of subnormal support." OFF )
 
+set( LIBCLC_NATIVECPU_FLAGS_X86_64 ""
+  CACHE STRING "Semicolon-separated list on compiler flags for x86_64 libclc target.")
```

Comment:
typo: `of`.

Also, will this need updating for AMD in #13911? Should we prepare for that somehow?